source/androidx.compose.foundation/foundation-layout-android/Additions/Additions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using Android.Runtime;
+using Java.Interop;
+
+namespace AndroidX.Compose.Foundation.Layout 
+{
+
+	// Metadata.xml XPath class reference: path="/api/package[@name='androidx.compose.foundation.layout']/class[@name='PaddingValues.Absolute']"
+	// [global::Android.Runtime.Register ("androidx/compose/foundation/layout/PaddingValues$Absolute", DoNotGenerateAcw=true)]
+	public sealed partial class PaddingValuesAbsolute //: global::Java.Lang.Object, global::AndroidX.Compose.Foundation.Layout.IPaddingValues 
```

Comment:
Need to remove or comment this out, as there are no C# bindings in this package.

Reply:
Removed